src/custom/state/operator/reducer.ts

```diff
@@ -0,0 +1,24 @@
+import { createReducer } from '@reduxjs/toolkit'
+import { updateTip, clearTip, SerializedToken, Tip } from './actions'
+
+type TipsMap = Record<SerializedToken['address'], { token: SerializedToken; tip: Tip }>
```

Comment:
SHould we give the params of the action a type (name) so it's easier to read and there's no repetition?